src/main/java/core/basesyntax/FigureBehaviour.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public interface FigureBehaviour {
+
+    String getName();
+
+    double getArea();
+
+    void printUniqueParameters();
```

Comment:
let's remove this method, instead create in each class unique method like getRadius()